ffi/jpeg.lua

```diff
@@ -0,0 +1,99 @@
+--[[--
+Module for JPEG decoding/encoding.
+
+Currently, this is a LuaJIT FFI wrapper for turbojpeg lib.
```

Comment:
"Currently" would just make me worry it's about to change. ;-)
```suggestion
This is a LuaJIT FFI wrapper for turbojpeg.
```

Reply:
You are right, razor sharp.